source/rust_verify/src/rust_to_vir_expr.rs

```diff
@@ -780,6 +835,199 @@ pub(crate) fn pattern_to_vir<'tcx>(
     Ok(pattern)
 }
 
+/// Check for the #[verifier(invariant_block)] attribute
+
```

Comment:
Very minor: usually there isn't a newline between the docstring and the function.